grakn-test-tools/src/main/java/ai/grakn/generator/AbstractOntologyConceptGenerator.java

```diff
@@ -112,26 +100,21 @@ private final boolean includeNonMeta() {
         return !meta.orElse(false);
     }
 
-    private final boolean includeAbstract(){
-        return includeAbstract.orElse(true);
-    }
-
     final AbstractOntologyConceptGenerator<T> excludeMeta() {
         meta = Optional.of(false);
         return this;
     }
 
-    final AbstractOntologyConceptGenerator<T> excludeAbstract() {
-        includeAbstract = Optional.of(false);
-        return this;
-    }
-
     public final void configure(Meta meta) {
-        this.meta = Optional.of(meta.value());
+        Preconditions.checkArgument(
+                !this.meta.isPresent() || this.meta.get(), "Cannot specify parameter is both meta and non-meta");
```

Comment:
Haha, just if you write a property test with parameter: `@Meta @NonMeta Concept concept` :P